### DISCUSSION TITLE
How does one get clickable urls for web articles under bibliography?

### DISCUSSION THREAD

**Discussion by Blakeinstein** [typst/typst#495] [Q&A]
I have a `URL` field under my web-articles in my `references.bib`.

In latex, these would render out as clickable urls but how does one get a similar result in Typst?

```bib
@web{tagged-blog,
  author = {AUTHOR},
  title = {TITLE},
  publisher = {Blog},
  URL = {https://SOME.URL},
  year = 2023
}
```

**Comment by Qottmann** [comment]
Yes, it is clickable. However there is always the "Available: " in front, which I would like to remove. Anybody know how?

**Reply by laurmaedje** [reply]
Better bibliography customization is next up on the roadmap. Until then you can work around it by adding `#show "Available: ": ""` in front of the bibliography.

**Reply by Qottmann** [reply]
Thanks!
